src/Elmish.WPF/ViewModelModule.fs

```diff
@@ -1,16 +1,7 @@
 ﻿module Elmish.WPF.ViewModel
 
-open Microsoft.Extensions.Logging.Abstractions
-
 /// Creates a design-time view model using the given model and bindings.
 let designInstance (model: 'model) (bindings: Binding<'model, 'msg> list) =
-  let args =
-    { initialModel = model
-      dispatch = ignore
-      loggingArgs =
-        { performanceLogThresholdMs = 1
-          nameChain = "main"
-          log = NullLogger.Instance
-          logPerformance = NullLogger.Instance } }
+  let args = ViewModelArgs.fake model
```

Comment:
This change is good.  Can you make it in its own PR so we can quickly merge it?

Reply:
See #460

Reply:
And #461